pro/src/components/IndividualOffer/PriceCategoriesScreen/PriceCategoriesScreen.tsx

```diff
@@ -336,17 +339,21 @@ export const PriceCategoriesScreen = ({
                       label="Intitulé du tarif"
                       description="Par exemple : catégorie 2, moins de 18 ans, pass 3 jours..."
                       maxLength={PRICE_CATEGORY_LABEL_MAX_LENGTH}
-                      count={field.price.toString().length}
+                      count={priceCategories[index]?.label.length}
                       className={styles['label-input']}
                       labelClassName={styles['label-input-label']}
                       disabled={priceCategories.length <= 1 || isDisabled}
+                      error={
+                        errors.priceCategories?.[index]?.label?.message ||
```

Comment:
```suggestion
                        errors.priceCategories?.[index]?.label?.message ??
```

C'est très micro et optionnel (j'ai l'habitude des règles de lint qui forcent l'utilisation plus safe de `??` partout où ce ne sont pas des booléens absolus)